types/react-map-gl/index.d.ts

```diff
@@ -215,7 +223,16 @@ export class LinearInterpolator extends TransitionInterpolator {
     constructor(transitionProps?: string[]);
 }
 
-export class FlyToInterpolator extends TransitionInterpolator {}
+export type FlyToInterpolatorProps = {
```

Comment:
Can we make this an interface instead? The rest of the types use interfaces when possible.

Reply:
Sure, Actually the builds were failing because I used 'type' instead of 'interface', just fixed it in a new commit.